llama_index/indices/knowledge_graph/base.py

```diff
@@ -45,8 +54,11 @@ def __init__(
         self,
         nodes: Optional[Sequence[Node]] = None,
         index_struct: Optional[KG] = None,
+        service_context: Optional[ServiceContext] = None,
+        storage_context: Optional[StorageContext] = None,
         kg_triple_extract_template: Optional[KnowledgeGraphPrompt] = None,
         max_triplets_per_chunk: int = 10,
+        graph_store: Optional[GraphStore] = None,
```

Comment:
nit: if graph store is already a part of the storage context, should we still accept it separately here?